reana_commons/openapi_specifications/reana_server.json

```diff
@@ -429,6 +429,26 @@
                     "slurmcern"
                   ]
                 },
+                "dask_cluster_default_number_of_workers": {
+                  "title": "Default number of workers for Dask clusters",
+                  "value": "2Gi"
+                },
+                "dask_cluster_default_single_worker_memory": {
+                  "title": "Amount of memory for one Dask worker by default",
+                  "value": "2Gi"
+                },
+                "dask_cluster_max_memory_limit": {
+                  "title": "Maximum memory limit for Dask clusters",
+                  "value": "16Gi"
+                },
+                "dask_cluster_max_single_worker_memory": {
+                  "title": "Maximum amount of memory for one Dask worker",
+                  "value": "8Gi"
+                },
+                "dask_enabled": {
+                  "title": "Dask workflows allowed in the cluster",
```

Comment:
Boolean to inform whether cluster supports running Dask workflows.

(BTW some wild thoughts on the necessity of this variable. I'm wondering whether we could use some of the other existing variables to decide whether Dask is enabled or disabled on the given deployment. OT1H, if the total memory limit is 0, and  the number of workers is 0, then this could in theory function as a sign that Dask is not supported. OTOH, we may want to start offering more enabled/disabled options for various functionalities in the future, such as is CWL enabled, is Snakemake enabled, etc. So I kind of like having it :wink: since we could generalise it towards enabling/disabling various functionalities in general. We can muse  more IRL.)